initial-configuration/jenkins/jenkins-docker/config.xml

```diff
@@ -220,12 +225,12 @@
             <comparator class="hudson.util.CaseInsensitiveComparator" reference="../../../../../../views/listView/jobNames/comparator"/>
           </default>
           <int>8</int>
+          <string>project_specific_override_repo</string>
+          <string>__PROJECT_SPECIFIC_OVERRIDE_REPO__</string>
```

Comment:
```suggestion
```
Will no longer be used in the latest version of the all-in-one. I do not believe we intend to maintain backwards compatibility.

Reply:
True I mean to delete that